### ISSUE TITLE
StripeException.getMessage adds additional info, making it impossible to get the "raw" message

### ISSUE THREAD

**Issue by jxtps** [stripe/stripe-java#1168]
[StripeException.getMessage](https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/exception/StripeException.java#L39) currently reads: 
```java
  public String getMessage() {
    String additionalInfo = "";
    if (code != null) {
      additionalInfo += "; code: " + code;
    }
    if (requestId != null) {
      additionalInfo += "; request-id: " + requestId;
    }
    return super.getMessage() + additionalInfo;
  }
```
Unfortunately, this makes it impossible to get the actual message *without* the code & request ID, and I need that. 

How about adding: 
```java
public String getUnderlyingMessage() { return super.getMessage(); } 
```

Thanks!

**Comment by remi-stripe** [comment]
@jxtps We're working on this feature in https://github.com/stripe/stripe-java/pull/1167 which we plan to merge after the holidays!

**Comment by richardm-stripe** [comment]
Fixed in #1167, released in [v20.34.0](https://github.com/stripe/stripe-java/blob/master/CHANGELOG.md#20340---2021-01-07)
